node/cli/src/service.rs

```diff
@@ -139,6 +137,8 @@ pub struct ServiceConfig {
     pub chain_spec_path: Option<PathBuf>,
     pub log_level: log::LevelFilter,
     pub rpc_port: u16,
+    /// whether to produce blocks with a set time period
+    pub batch_transactions: bool,
```

Comment:
I don't think it is a good idea to mix DevNet-specific and general configs. This needs to be separated. I suggest we always batch transactions for the DevNet, which means we do not need this flag.

Reply:
Tests will fail if we batch transactions

Reply:
The current mode of running the DevNet is with batched transactions. If we add a boolean command flag that is only used to trigger a different execution path for tests then the tests will not be testing the real execution path. In general, adding a boolean flag just to hack around the tests is not okay.